hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AtttributeResolver.kt

```diff
@@ -15,10 +15,32 @@ typealias Attributes = Map<String, Attribute>
 
 @Configuration
 class AttributeResolver {
+  // attributes contains everything you can use in queries in the attributes search
   @Bean
   fun attributes(): Attributes = getAttributes(Prisoner::class)
+
+  // attributesAndObjects contains everything you can request to be populated on the Prisoner record
+  @Bean
+  fun attributeAndObjects(attributes: Attributes): List<String> = attributes.attributeAndObjects()
```

Comment:
We have an endpoint that exposes the contents of `attributes()` to tell people using attributes search which attributes they can query over.

We'll need a similar endpoint exposing `attributesAndObjects()` to tell people which attributes they can request are returned from the Prisoner record.